full-service/src/service/wallet_impl.rs

```diff
@@ -57,39 +58,43 @@ pub struct WalletService<
     T: BlockchainConnection + UserTxConnection + 'static,
     FPR: FogPubkeyResolver + Send + Sync + 'static,
 > {
-    wallet_db: WalletDb,
+    pub wallet_db: WalletDb,
     ledger_db: LedgerDB,
     peer_manager: McConnectionManager<T>,
     network_state: Arc<RwLock<PollingNetworkState<T>>>,
     fog_pubkey_resolver: Option<Arc<FPR>>,
     _sync_thread: SyncThread,
     /// Monotonically increasing counter. This is used for node round-robin selection.
     submit_node_offset: Arc<AtomicUsize>,
-    logger: Logger,
+    offline: bool,
+    pub logger: Logger,
```

Comment:
Whats accessing this externally?

Reply:
Here's an example where the trait impl is using it: https://github.com/mobilecoinofficial/full-service/blob/880f94b73ffd8c4fb846430d1011367cc8758c76/full-service/src/service/password_manager.rs#L126